accurest-core/src/main/groovy/io/codearte/accurest/builder/SpockMethodBodyBuilder.groovy

```diff
@@ -122,6 +122,17 @@ abstract class SpockMethodBodyBuilder {
 		return trimRepeatedQuotes(json)
 	}
 
+	protected Map<String, Object> getMultipartParameters() {
+		return (Map<String, Object>)request.multipart.serverValue
```

Comment:
so far the only usage is conditioned with if (request.multipart), but I guess some "?."s won't hurt